kong/plugins/acl/api.lua

```diff
@@ -53,5 +53,31 @@ return {
     DELETE = function(self, dao_factory)
       crud.delete(self.acl, dao_factory.acls)
     end
+  },
+  ["/acls"] = {
+    GET = function(self, dao_factory)
+      crud.paginated_set(self, dao_factory.acls)
+    end
+  },
+  ["/acls/:acl_id/consumer"] = {
+    before = function(self, dao_factory, helpers)
+      local filter_keys = {
+       id = self.params.acl_id
+      }
+
+      local acls, err = dao_factory.acls:find_all(filter_keys)
+      if err then
+        return helpers.yield_error(err)
+      elseif next(acls) == nil then
+        return helpers.responses.send_HTTP_NOT_FOUND()
+      end
+
+      self.params.username_or_id = acls[1].consumer_id
```

Comment:
I think that we are missing a `self.params.acl_id = nil` instruction here. Not an issue with `find_consumer_by_username_or_id`, but it could become one further down the road in one of the HTTP verb handlers. Better safe than sorry!

I will take care of this while merging, no worries.